storage/mysql/storage_test.go

```diff
@@ -616,6 +634,34 @@ func TestGetLeavesByHash(t *testing.T) {
 	checkLeafContents(leaves[0], sequenceNumber, dummyRawHash, dummyHash, data, t)
 }
 
+func TestGetLeavesByLeafValueHash(t *testing.T) {
+	// Create fake leaf as if it had been sequenced
+	logID := createLogID("TestGetLeavesByLeafValueHash")
+	db := prepareTestLogDB(logID, t)
+	defer db.Close()
+
+	data := []byte("some data")
+
+	createFakeLeaf(db, logID.logID, dummyRawHash, dummyHash, data, sequenceNumber, t)
+
+	s := prepareTestLogStorage(logID, t)
+	tx := beginLogTx(s, t)
+	defer tx.Commit()
+
+	hashes := [][]byte{dummyRawHash}
+	leaves, err := tx.GetLeavesByLeafValueHash(hashes, false)
+
+	if err != nil {
```

Comment:
(style) As above; also, Go tends towards table-driven tests -- it would be good to combine this with the previous test into a table, if possible.